src/main/kotlin/model/PrepareCaseForCourt.kt

```diff
@@ -1,33 +1,113 @@
 package uk.gov.justice.hmpps.architecture
 
+import com.structurizr.model.Container
+import com.structurizr.model.Location
 import com.structurizr.model.Model
 import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.AutomaticLayout
 import com.structurizr.view.ViewSet
 
 class PrepareCaseForCourt private constructor() {
   companion object : HMPPSSoftwareSystem {
     lateinit var system: SoftwareSystem
+    lateinit var prepareCaseUI: Container
+    lateinit var courtCaseService: Container
+    lateinit var courtCaseMatcher: Container
 
     override fun defineModelEntities(model: Model) {
-      system = model.addSoftwareSystem(
-        "Prepare a Case for Court",
-        """
-        Service for Probation Officers working in magistrates' courts, providing them with a
-        single location to access the defendant information they need to provide sound and
-        timely sentencing guidance to magistrates
-        """.trimIndent()
-      )
+
+      system = model.addSoftwareSystem("Prepare a Case for Court", "Digital Service for Probation Officers working in magistrates' courts, " +
+        "providing them with a single location to access the defendant information they need to provide sound and timely sentencing guidance to magistrates")
+        .apply {
+          setLocation(Location.Internal)
+        }
+
+      val casesDb = system.addContainer(
+        "Cases Database",
+        "Holds court lists, cases and offender ids",
+        "PostgreSQL"
+      ).apply {
+        Tags.DATABASE.addTo(this)
+        Tags.SOFTWARE_AS_A_SERVICE.addTo(this)
```

Comment:
What do you think about replacing this SaaS tag with `CloudPlatform.rds.add(this)`? The deployment means would imply that it's a managed service without extra tagging needed.